test/Pass/insert-after-phis.ll

```diff
@@ -0,0 +1,46 @@
+; RUN: %opt -load %pass -souper -S -stats -o - %s 2>&1 | %FileCheck %s
```

Comment:
I was having a bit of a hard time coming up with good tests. If I could tell souper to just run once, I think I could create a better test (that checks we replace a PHI instruction with an add, for example). 

Other than that, I can't come up with an input program where the PHI node is replaced with a non-phi node, and the whole thing isn't optimized away at some point later.